apps/mobile/src/components/Button.tsx

```diff
@@ -157,7 +157,8 @@ export function Button({
 
   const [active, setActive] = useState(false);
 
-  const variantClassNames = buttonVariants[colorScheme][variant][active ? 'active' : 'inactive'];
+  const variantClassNames =
+    buttonVariants[colorScheme][disabled ? 'disabled' : variant][active ? 'active' : 'inactive'];
```

Comment:
this way we don't have to manually set the variant to `disabled` when the button is in a disabled state